cmd/fixture_utils/step_runner.go

```diff
@@ -78,6 +78,60 @@ func WaitForNextBlockWithErrorCheck(t *testing.T) {
 	}
 }
 
+// RunCreateAccount is a function to create account
+func RunCreateAccount(step FixtureStep, t *testing.T) {
+	if step.ParamsRef != "" {
+		caKey := GetAccountKeyFromTempName(step.ParamsRef, t)
+		localKeyResult, err := inttest.AddNewLocalKey(caKey)
+		t.WithFields(testing.Fields{
+			"key":              caKey,
+			"local_key_result": localKeyResult,
+		}).MustNil(err, "error creating local Key")
+		t.Log("localKeyResult, err :=", localKeyResult, err)
+		result, logstr, err := inttest.CreateChainAccount(caKey)
+		t.WithFields(testing.Fields{
+			"result": result,
+			"logstr": logstr,
+		}).MustNil(err, "error creating account on chain")
+		WaitForNextBlockWithErrorCheck(t)
+		WaitForNextBlockWithErrorCheck(t)
```

Comment:
We have a function to wait 2 blocks and it won't be necessary to use this function 2 times.

Reply:
It's modified